rebasehelper/cli.py

```diff
@@ -54,112 +87,121 @@ def add_args(self):
             "--verbose",
             default=False,
             action="store_true",
-            help="Output is more verbose (recommended)"
+            help="be more verbose (recommended)"
         )
         self.parser.add_argument(
             "-p",
             "--patch-only",
             default=False,
             action="store_true",
-            help="Only apply patches"
+            help="only apply patches"
         )
         self.parser.add_argument(
             "-b",
             "--build-only",
             default=False,
             action="store_true",
-            help="Only build SRPM and RPMs"
+            help="only build SRPM and RPMs"
         )
         self.parser.add_argument(
             "--buildtool",
+            # FIXME: set of choices could be obtained dynamically
+            choices={"mock", "rpmbuild", "koji", "copr"},
             default="mock",
-            help="Select the build tool [mock, rpmbuild, koji, copr]. 'mock' is used by default."
+            help="build tool to use, defaults to %(default)s"
         )
         self.parser.add_argument(
             "--pkgcomparetool",
-            default=False,
-            help="Select the tool for comparing two packages [pkgdiff, rpmdiff, abipkgdiff, csmock]. All compare tools"
-                 " are run by default."
+            # FIXME: set of choices could be obtained dynamically
+            choices={"pkgdiff", "rpmdiff", "abipkgdiff", "csmock"},
+            default={"pkgdiff", "rpmdiff", "abipkgdiff"},
+            type=lambda s: set(s.split(',')),
```

Comment:
I was thinking more about using action 'append' and allow user to use the argument more than once, but I'm OK also with your solution.